packages/kernel-5.15/6001-lustre-fix-Werror-enum-int-mismatch.patch

```diff
@@ -0,0 +1,35 @@
+From 82032af947c7a8b17ec30483a6d4a2d44759561f Mon Sep 17 00:00:00 2001
```

Comment:
Is this a patch that should be upstreamed to the Amazon Linux kernel?

Reply:
> Is this a patch that should be upstreamed to the Amazon Linux kernel?

Not sure that they'll care until they move to a newer GCC for kernel builds, but I've mentioned it.